### DISCUSSION TITLE
How to create entry points to match source tree

### DISCUSSION THREAD

**Discussion by rnarcos** [webpack/webpack#14044] [Q&A]
I have a library, which I'm trying to create a bundle that would follow the same principles as the material-ui project.

I'm trying to enable multiple imports strategy like:

```javascript
import { Button } from 'my-library';
import { Button } from 'my-library/components';`
import Button from 'my-library/components/Button';
import Button from 'my-library/components/Button/Button.component.jsx';
```

I am not entirely sure how would I be able to copy the same file-tree structure, while also deep-bundling my library. Or if I should be using webpack or other mechanism to achieve this. Have you ever had to deal with this using webpack?

**Comment by alexander-akait** [comment]
It is not supported yet, in our roadmap

**Reply by rnarcos** [reply]
Thank you for your answer! Would you know how could I approach this without webpack? @alexander-akait

**Reply by alexander-akait** [reply]
Do not bundle, all builders support ECMA modules, even Node.js supports them, so you don't need to bundle

**Comment by rnarcos** [comment]
The problem is that I am creating a react library, so I need to use at least babel to transpile the code into a browser-valid javascript. You're saying that I should only copy my src folder into my dist folder?

**Reply by alexander-akait** [reply]
Just use `babel` and nothing more
